src/assets/toolkit/styles/utils/height.css

```diff
@@ -0,0 +1,11 @@
+/* Set height limits with vh units */
+
+.u-limitHeight75 {
+  max-height: 75vh !important;
+}
+
+@media (min-height: 550px) {
+  .u-mdvh-limitHeight50 {
```

Comment:
I'm not sure `vh` is the right addition, because the media query itself has nothing to do with `vh` (only the rule that's applied).

I'd be fine with `mdh`. It's not _great_, but it's only one more character.

We should probably define the media query where the others ones are in our theme, though.